src/styles/molecules/_tabs.scss

```diff
@@ -0,0 +1,34 @@
+.tab{
+  width: 40%;
+  display: inline-block;
+}
+.tab__nav ul{
+  margin-bottom: 0px;
+  padding: 0;
+  display: flex;
+  justify-content: space-between;
+}
+.tab__item{
+  text-align: center;
+  width: 29%;
```

Comment:
This will only work if you have three tabs. If you instead just remove `width` and use `flex: 1` it will automatically distribute the width. It shouldn't have a `display: inline-block` either :)